schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/activity/DefaultActivityService.java

```diff
@@ -493,6 +256,7 @@ public Key addActivityPackage(ActivityPackage activityPackage, List<Activity> ac
         TransactionOperator.executeNoEx(new ModelOperation<Void>() {
             @Override
             public Void execute(Transaction tx) throws ModelException {
+                activityPackageDao.save(activityPackage, organizationId);
                 Datastore.put(tx, models);
                 tx.commit();
```

Comment:
do we not have a `ActivityPackageActivityDao`?

if YES we should use it here
else we should use it here

:-)